Unable to activate extension: Cannot read property 'major' of null

When trying to activate extension I get the following error: Cannot read property 'major' of null".

## Description
I tried the "Retry activation" button and also uninstall/reinstall the extension but I got the same error.

## Steps to Reproduce
1. Install IBM Blockchain Platform extension from VS Code 
2. Go to IBM Blockchain Platform in the VS Code Side Bar 

## Log File
[17/04/2020 à 18:18:10] [INFO] Starting IBM Blockchain Platform Extension
[17/04/2020 à 18:18:10] [INFO] Attempting to require dependency: grpc
[17/04/2020 à 18:18:10] [INFO] Restoring command registry
[17/04/2020 à 18:18:11] [INFO] Registering commands
[17/04/2020 à 18:18:12] [INFO] Execute stored commands in the registry
[17/04/2020 à 18:18:12] [INFO] IBM Blockchain Platform Extension activated
[17/04/2020 à 18:18:12] [ERROR] Failed to activate extension: TypeError: Cannot read property 'major' of null
[17/04/2020 à 18:18:12] [ERROR] TypeError: Cannot read property 'major' of null
	at Function.<anonymous> (c:\Users\xxx\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.26\build\extension\util\ExtensionUtil.js:596:54)
	at Generator.next (<anonymous>)
	at fulfilled (c:\Users\xxx\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.26\build\extension\util\ExtensionUtil.js:18:58)

## Your Environment
* IBM Blockchain Platform Extension Version: 1.0.26
* Visual Studio Code version: 1.44.2
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1): Node.js 10.20.1, npm 6.14.4
* Operating System and version: Windows 10.0.18363

When I install the extension I got also this error:

[20/04/2020 à 12:16:51] [INFO] Error requiring dependency: Failed to load gRPC binary module because it was not installed for the current system
Expected directory: electron-v7.1-win32-x64-unknown
Found: [node-v64-linux-x64-glibc]
This problem can often be fixed by running "npm rebuild" on the current system
Original error: Cannot find module 'c:\Users\xxx\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.26\node_modules\grpc\src\node\extension_binary\electron-v7.1-win32-x64-unknown\grpc_node.node'

Perhaps is it related? I don't how to fix this.

--- a/packages/blockchain-extension/extension/util/ExtensionUtil.ts
+++ b/packages/blockchain-extension/extension/util/ExtensionUtil.ts
@@ -659,7 +659,7 @@ export class ExtensionUtil {
         // This needs to be done as a seperate call to make sure the dependencies have been installed
         const generatorVersion: string = dependencies['generator-fabric'];
 
-        if (extensionData.generatorVersion !== null && generatorVersion !== extensionData.generatorVersion) {
+        if (extensionData.generatorVersion && generatorVersion !== extensionData.generatorVersion) {
             // If the latest generator version is not equal to the previous used version
 
             let teardownRuntimes: boolean = false;
@@ -744,7 +744,7 @@ export class ExtensionUtil {
             await vscode.commands.executeCommand('setContext', 'local-fabric-enabled', false);
         }
 
-        if (extensionData.generatorVersion === null) {
+        if (!extensionData.generatorVersion) {
             extensionData.generatorVersion = generatorVersion;
             await GlobalState.update(extensionData);
         }

--- a/packages/blockchain-extension/test/util/ExtensionUtil.test.ts
+++ b/packages/blockchain-extension/test/util/ExtensionUtil.test.ts
@@ -1720,11 +1720,11 @@ describe('ExtensionUtil Tests', () => {
             });
         });
 
-        it(`shouldn't update the generator version to latest when null`, async () => {
+        it(`shouldn't update the generator version to latest when undefined`, async () => {
             await vscode.workspace.getConfiguration().update(SettingConfigurations.HOME_SHOW_ON_STARTUP, true, vscode.ConfigurationTarget.Global);
 
             globalStateGetStub.returns({
-                generatorVersion: null
+                generatorVersion: undefined
             });
 
             executeCommandStub.resolves();

